pkg/util/versions/versions.go

```diff
@@ -0,0 +1,43 @@
+package versions
+
+import (
+	"os"
+
+	"github.com/Masterminds/semver/v3"
+	"sigs.k8s.io/yaml"
+)
+
+const previewVersion = "preview"
+
+func Supported(path string) ([]*semver.Version, error) {
```

Comment:
Naming is a bit weird here. It's hard to guess what this function does by the name.
It has nothing to do with supported versions, no?

It's more of `ParseFromFile`?

Reply:
ah right, some of them are out of support, I'll rename the func